File: Day-02/Python/dice_roller.py
import random
import time

choice = 'y'

#Printing the Output
def show_Dice(random_number):

    def one(random_number):
    
        print("You Got {}!".format(random_number))
        print(" _______ ")
        print("|       |")
        print("|   0   |")
        print("|_______|")
        return 0
    
    def two(random_number):
    
        print("You Got {}!".format(random_number))
        print(" _______ ")
        print("|       |")
        print("| 0   0 |")
        print("|_______|")
        return 0

    def three(random_number):
    
        print("You Got {}!".format(random_number))
        print(" _______ ")
        print("|   0   |")
        print("| 0   0 |")
        print("|_______|")
        return 0
    
    def four(random_number):
    
        print("You Got {}!".format(random_number))
        print(" _______ ")
        print("| 0   0 |")
        print("| 0   0 |")
        print("|_______|")
        return 0

    def five(random_number):
    
        print("You Got {}!".format(random_number))
        print(" _______ ")
        print("| 0   0 |")
        print("|   0   |")
        print("|_0___0_|")
        return 0

    def six(random_number):
    
        print("You Got {}!".format(random_number))
        print(" _______ ")
        print("| 0   0 |")
        print("| 0   0 |")
        print("|_0___0_|")
        return 0

    if(random_number == 1):    one(random_number)
    elif(random_number == 2):  two(random_number)
    elif(random_number == 3):  three(random_number)
    elif(random_number == 4):  four(random_number)
    elif(random_number == 5):  five(random_number)
    else:   six(random_number)

    return 0

#Welcome screen!
print()
print("\tRolling Dice!")
print("\n\t ....")
print()
time.sleep(3)

#Rolling a Dice!
while(choice == 'y' or choice =='Y'):
    random_number = random.randint(1,6)
    show_Dice(random_number)
    choice = input("\nRoll Again ? (y/n) :")
    if(choice == 'y'or choice == 'Y'):
        print("\nRolling...")
        time.sleep(3)
    
print("\n\tThank You for Playing! :)\n")

File: Day-02/Java/diceRoller.java
import java.io.*;
import java.util.*;

class diceRoller {

    static int number;
    static char choice;

    public static void main(String[] args){
    
        Scanner scan = new Scanner(System.in);
        System.out.println();
        System.out.println("\tRolling Dice!");

        do {
            rollDice();
            System.out.println();
            showDice();

            //Getting Choice
            System.out.print("\nRoll Again ? (y/n): ");
            choice = scan.next().charAt(0);
        }while(choice == 'y' || choice == 'Y');
        System.out.println("\n\tThank You for Playing! :)\n");
    }
    
    //Rolling Dice
    static void rollDice () {
        number = 1 + (int)(Math.random() * 6);
    }

    //Display Output
    static void showDice() {
        switch(number) {
            case 1:
                System.out.println("You Got "  + number + "!\t\t");
                System.out.println(" _______ ");
                System.out.println("|       |");
                System.out.println("|   0   |");
                System.out.println("|_______|");
                break;
            case 2:
                System.out.println("You Got "  + number + "!\t\t");
                System.out.println(" _______ ");
                System.out.println("|       |");
                System.out.println("| 0   0 |");
                System.out.println("|_______|");
                break;
            case 3:
                System.out.println("You Got "  + number + "!\t\t");
                System.out.println(" _______ ");
                System.out.println("|   0   |");
                System.out.println("| 0   0 |");
                System.out.println("|_______|");
                break;
            case 4:
                System.out.println("You Got "  + number + "!\t\t");
                System.out.println(" _______ ");
                System.out.println("| 0   0 |");
                System.out.println("|       |");
                System.out.println("|_0___0_|");
                break;
            case 5:
                System.out.println("You Got "  + number + "!\t\t");
                System.out.println(" _______ ");
                System.out.println("| 0   0 |");
                System.out.println("|   0   |");
                System.out.println("|_0___0_|");
                break;
            case 6:
                System.out.println("You Got "  + number + "!\t\t");
                System.out.println(" _______ ");
                System.out.println("| 0   0 |");
                System.out.println("| 0   0 |");
                System.out.println("|_0___0_|");
                break;
        }
    }
}